ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -487,20 +487,18 @@ public Builder applyPraterNetworkDefaults() {
               "enr:-LK4QLM_pPHa78R8xlcU_s40Y3XhFjlb3kPddW9lRlY67N5qeFE2Wo7RgzDgRs2KLCXODnacVHMFw1SfpsW3R474RZEBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpB53wQoAAAQIP__________gmlkgnY0gmlwhANBY-yJc2VjcDI1NmsxoQNsZkFXgKbTzuxF7uwxlGauTGJelE6HD269CcFlZ_R7A4N0Y3CCI4yDdWRwgiOM");
     }
 
-    public Builder applyKintsugiNetworkDefaults() {
+    public Builder applyRopstenNetworkDefaults() {
       return reset()
-          .constants(KINTSUGI.configName())
+          .constants(ROPSTEN.configName())
           .startupTimeoutSeconds(120)
-          .eth1DepositContractDeployBlock(0)
+          .eth1DepositContractDeployBlock(12269949)
           .defaultInitialState(
-              "https://github.com/eth-clients/merge-testnets/raw/3d07b2914747c4a8bd847a27c272e048cb2d29d8/kintsugi/genesis.ssz")
+              "https://github.com/eth-clients/merge-testnets/raw/5cbccd4449b5cbfcd677d1ee13515671e0f03928/ropsten-beacon-chain/genesis.ssz")
```

Comment:
PR was merged: https://github.com/eth-clients/merge-testnets/pull/9

```suggestion
              "https://github.com/eth-clients/merge-testnets/raw/5b1b44aa912dd3433ba30d381345659c53918955/ropsten-beacon-chain/genesis.ssz")
```